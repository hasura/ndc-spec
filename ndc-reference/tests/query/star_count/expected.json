[
  {
    "rows": [
      {
        "articles_aggregate": {
          "rows": {
            "aggregates": {
              "count": 1
            }
          }
        },
        "last_name": {
          "value": "Landin"
        },
        "first_name": {
          "value": "Peter"
        }
      },
      {
        "articles_aggregate": {
          "rows": {
            "aggregates": {
              "count": 2
            }
          }
        },
        "last_name": {
          "value": "Hughes"
        },
        "first_name": {
          "value": "John"
        }
      }
    ]
  }
]