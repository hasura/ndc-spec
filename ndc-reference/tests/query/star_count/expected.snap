---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/star_count/request.json
---
[
  {
    "rows": [
      {
        "first_name": "Peter",
        "last_name": "Landin",
        "articles_aggregate": {
          "aggregates": {
            "count": 1
          }
        }
      },
      {
        "first_name": "John",
        "last_name": "Hughes",
        "articles_aggregate": {
          "aggregates": {
            "count": 2
          }
        }
      }
    ]
  }
]
